sdk/appconfiguration/app-configuration/src/appConfigCredential.ts

```diff
@@ -47,7 +47,7 @@ export class AppConfigCredential implements ServiceClientCredentials {
     webResource.headers.set("x-ms-content-sha256", contentHash);
     webResource.headers.set(
       "Authorization",
-      `HMAC-SHA256 Credential=${this.credential}, SignedHeaders=${signedHeaders}, Signature=${signature}`
+      `HMAC-SHA256 Credential=${this.credential}&SignedHeaders=${signedHeaders}&Signature=${signature}`
```

Comment:
This looks right as per the syntax https://docs.microsoft.com/en-us/azure/azure-app-configuration/rest-api-authentication-hmac#syntax

Reply:
And seems to have fixed the test failures at https://github.com/Azure/azure-sdk-for-js/pull/17583

Reply:
@HarshaNalluru Ok, I have canceled this modification and update the code to the original.

Reply:
Thanks for all your efforts and investigations @zzhxiaofeng 🐱‍💻